code/game/response_team.dm

```diff
@@ -30,13 +30,59 @@ var/can_call_ert
 		switch(alert("The station is not in red alert. Do you still want to dispatch a response team?",,"Yes","No"))
 			if("No")
 				return
+
 	if(send_emergency_team)
 		to_chat(usr, "<span class='warning'>Looks like somebody beat you to it!</span>")
 		return
 
+	var/customtext = sanitize(input(usr, "Please enter anything you want. Anything. Serious.", "What?", "") as message|null, MAX_PAPER_MESSAGE_LEN, extra = FALSE)
+	var/customname = sanitize_safe(input(usr, "Pick a title for the report.", "Title") as text|null)
+	if(!customtext)
+		customtext = "It would appear that an emergency response team was requested for [station_name()]. We will prepare and send one as soon as possible."
+	if(!customname)
+		customname = "Central Command"
+
 	message_admins("[key_name_admin(usr)] is dispatching an Emergency Response Team.", 1)
 	log_admin("[key_name(usr)] used Dispatch Response Team.")
-	trigger_armed_response_team(1)
```

Comment:
Вот эта функция делала больше  чем то, на что ты её заменил.

Reply:
Расчеты? Лок ЕРТ для консоли глав?

Reply:
```
/proc/trigger_armed_response_team(force = 0)
	if(!can_call_ert && !force) // Это для отличия консоли от педального верба
		return
	if(send_emergency_team) // Это проверка, отправляется ли сейчас ЕРТ
		return

	// Это шансы для консоли глав
	var/send_team_chance = ert_base_chance // Is incremented by increment_ert_chance.
	send_team_chance += 2*percentage_dead() // the more people are dead, the higher the chance
	send_team_chance += percentage_antagonists() // the more antagonists, the higher the chance
	send_team_chance = min(send_team_chance, 100)

	// Педальный верб
	if(force) send_team_chance = 100

	// Если шанс не прокнул - отказ
	if(!prob(send_team_chance))
		command_alert("It would appear that an emergency response team was requested for [station_name()]. Unfortunately, we were unable to send one at this time.", "Central Command", "noert")
		can_call_ert = 0 // Only one call per round, ladies.
		return
	// Если же прокнул - отправляем
	command_alert("It would appear that an emergency response team was requested for [station_name()]. We will prepare and send one as soon as possible.", "Central Command", "yesert")
	can_call_ert = 0 // Only one call per round, gentleman.

	// Сам функционал
	send_emergency_team = 1
	sleep(600 * 5)
	send_emergency_team = 0 // Can no longer join the ERT.
```

Reply:
Ну к примеру выставляла can_call_ert.

Ну и да, можно было бы переделать эту функцию чтобы не было "лишних расчётов"(проверка форсед - если не форсед, расчитываем и смотрим шанс - в ином случае забиваем), а так в итоге функция дублирующая(ещё и не полностью) механ существующей.

Reply:
Так `can_call_ert` игнорируется педальным вербом
`if(!can_call_ert && !force)`

Reply:
Так вызовом из консоли не игнорируется.

Reply:
`can_call_ert` нужен исключительно 
```
/obj/machinery/keycard_auth/proc/trigger_event()
	switch(event)
		if("Red alert")
			set_security_level(SEC_LEVEL_RED)
			feedback_inc("alert_keycard_auth_red",1)
		if("Grant Emergency Maintenance Access")
			make_maint_all_access(TRUE)
			feedback_inc("alert_keycard_auth_maintGrant",1)
		if("Revoke Emergency Maintenance Access")
			if(timer_maint_revoke_id)
				deltimer(timer_maint_revoke_id)
				timer_maint_revoke_id = 0
			revoke_maint_all_access(TRUE)
			feedback_inc("alert_keycard_auth_maintRevoke",1)
		if("Emergency Response Team")
			if(is_ert_blocked())
				to_chat(usr, "<span class='warning'>All emergency response teams are dispatched and can not be called at this time.</span>")
				return

			trigger_armed_response_team(1) // Вообще тут сто процентов был раньше 0
			feedback_inc("alert_keycard_auth_ert",1)
```

Reply:
Консоль уже давно не может вызвать ЕРТ, это убрано

Reply:
```
can_call_ert = 0 // Only one call per round, gentleman.
```

Я про вот эту строчку, если что.

Reply:
Поправка, в конфиге сервера выключена ЕРТ у глав:
```
if(!config.ert_admin_call_only)
			dat += "<li><A href='?src=\ref[src];triggerevent=Emergency Response Team'>Emergency Response Team</A></li>"
```

Reply:
Ну тогда в чём смысл старую функцию вообще сохранять, и эту конфиг. опцию? У тебя очень странный аргумент.

"Эта функция не делает расчёты которые нужны для вызоа от глав"
"Твоя функция перестала запрещать главам вызывать ЕРТ(повторно)"
"Так главы и не могут"

Reply:
Да, я понял претензию. Добавлю эту строчку